pkg/chunk/cassandra/storage_client.go

```diff
@@ -63,6 +64,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.DisableInitialHostLookup, "cassandra.disable-initial-host-lookup", false, "Instruct the cassandra driver to not attempt to get host info from the system.peers table.")
 	f.BoolVar(&cfg.SSL, "cassandra.ssl", false, "Use SSL when connecting to cassandra instances.")
 	f.BoolVar(&cfg.HostVerification, "cassandra.host-verification", true, "Require SSL certificate validation.")
+	f.StringVar(&cfg.HostSelectionPolicy, "cassandra.host-selection-policy", "RoundRobin", "Policy for selecting Cassandra host. Supported values are: RoundRobin, TokenAware.")
```

Comment:
**nit:** I'm not crazy about the config values requiring capitalization. Would it be possible to rename the config values to be `round-robin` and `token-aware`? Or is there an existing convention that would make the current values preferable?

Reply:
I chose the config values simply because I didn't find similar `StringVar` for reference. We can avoid capitalization.